quiz/index.html

```diff
@@ -5,11 +5,37 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>謎解きクイズ</title>
         <link rel="stylesheet" href="../globalstyle/buttonstyle.css" />
+        <link rel="stylesheet" href="./style/color.css" />
         <link rel="stylesheet" href="./style/style.css" />
+        <style>
+            .infomation {
+                text-align: center;
+            }
+            .infomation p {
+                font-size: 1.5rem;
+            }
+        </style>
     </head>
     <body>
         <div class="returnButton" onclick="returnHome()"><img src="../img/returnButton.png" /></div>
+        <div class="circleBox" onClick="clickCenterButton()">
+            <svg class="buttonCircle red" width="160" height="160">
+                <circle cx="80" cy="80" r="80" stroke-width="1"></circle>
+            </svg>
+        </div>
+        <div class="gameMessage">
+            <p class="red">Click Center Button</p>
+        </div>
+        <div class="infomation">
+            <p>真ん中のボタンを押して次のステージに進もう！</p>
+        </div>
     </body>
-    <script src="../globalscript/returnHome.js"></script>
+    <script src="../globalsrc/returnHome.js"></script>
     <script src="./src/main.js"></script>
+    <script>
+        function clickCenterButton() {
+            document.querySelector(".circleBox").classList.add("popOutAnimation");
```

Comment:
`querySelector` よりも `getElementByClassName`の方が最適です。  `querySelector`は無闇に使うな馬鹿野郎
```suggestion
            document.getElementByClassName("circleBox")[0].classList.add("popOutAnimation");
```

Reply:
`getElementsByClassName()` にしました
https://developer.mozilla.org/ja/docs/Web/API/Document/getElementsByClassName